events/d_2_data_agony.toml

```diff
@@ -72,21 +72,21 @@ This track will cover pain points stemming from the way data is structured withi
 # - use 15m breaks, one in the morning, one or two in the afternoon
 [[timeslots]]
 startTime="09:00"
-speakers=["@speaker_name", "@other_speaker_name"]
+speakers=["@aschmahmann"]
 title="Welcome standup discussion"
-description=""
+description="What's your vision for working with data in IPFS, and what about how it works today causes you the most agony?"
 
 [[timeslots]]
 startTime="09:30"
-speakers=["@yourname"]
+speakers=["@aschmahmann"]
 title="Track Intro"
-description=""
+description="Why are we here and what do we want to get out of today?"
 
 [[timeslots]]
 startTime="10:00"
-speakers=[]
-title="Open Slot"
-description=""
+speakers=["TBD"]
+title="Dealing with block limits"
```

Comment:
Left this as TBD for now. I think @dignifiedquire has a couple of areas he may want to talk about and this could be one.

I also have a proposal for a solution here (basically https://github.com/protocol/beyond-bitswap/pull/29) that I'd like to get feedback on and see if people are reasonably happy with as a solution to this problem.